zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -75,6 +75,13 @@ private[consumer] final class Runloop private (
   private[internal] def removeSubscription(subscription: Subscription): UIO[Unit] =
     commandQueue.offer(RunloopCommand.RemoveSubscription(subscription)).unit
 
+  private[internal] def stopSubscribedTopicPartitions(subscription: Subscription): UIO[Unit] =
```

Comment:
This method is stopping streams, not subscriptions. WDYT of:

```suggestion
  private[internal] def stopStreams(subscription: Subscription): UIO[Unit] =
```